ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+set -x
+
+cp ${CLUSTER_PROFILE_DIR}/pull-secret /tmp/pull-secret
+oc registry login --to /tmp/pull-secret
+ocp_version=$(oc adm release info --registry-config /tmp/pull-secret ${RELEASE_IMAGE_LATEST} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
+ocp_major_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $1}' )
+ocp_minor_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $2}' )
+rm /tmp/pull-secret
+
+
+function remove_secrets()
+{
+    local path="$1"
+    local sec_namespace="$2"
+    if [ ! -e "${path}" ]; then
+        echo "[ERROR] ${path} does not exist"
+        return 2
+    fi
+    pushd "${path}"
+    for i in *.yaml; do
+        res_kind=$(yq-go r "${i}" 'kind')
+        res_namespace=$(yq-go r "${i}" 'metadata.namespace')
+        if [[ "${res_kind}" == "Secret" ]] && [[ "${res_namespace}" == "${sec_namespace}" ]]; then
+            echo "[WARN] Remove file ${i} which matches \"${sec_namespace}\""
+            rm -f "${i}"
+            [ $? -ne 0 ] && echo "[ERROR] error remove file ${i}" && return 1
+        fi
+    done
+    popd
+    return 0
+}
+
+if (( ocp_minor_version == 12 && ocp_major_version == 4 )); then
+    # 4.12, if Storage operator not installed, then remove CSI driver CR
+    if [[ "${BASELINE_CAPABILITY_SET}" == "None" ]] && [[ ! "${ADDITIONAL_ENABLED_CAPABILITY_SET}" =~ "Storage" ]]; then
```

Comment:
v4.11 set is {baremetal, openshift-samples, marketplace}.
If ${BASELINE_CAPABILITY_SET} set to `v4.11`, no additionalCapabilitySet defined or not including storage co,  storage is also disabled.